### PR TITLE
[issue-15]: stat -x is only availabe in macOS

### PR DESCRIPTION
**Pull request by cht8687**

### CODE REVIEW

**Comment by stevemao** [comment]
Is there a command that works on Linux?

**Comment by cht8687** [comment]
updated to keep consistent...

File: readme.md [hunk 1]
```diff
@@ -145,13 +145,8 @@ mkdir -p 'path/may/not/exist/untitled folder'
...
 ```
-stat -x readme.md
-```
-
-or
-
-```
-du -sh readme.md
+stat -x readme.md  # on macOS
+du -sh readme.md   # on Linux
```
Comment (stevemao) [inline readme.md]: just tested, `du -sh readme.md` also works on macOS

**Comment by stevemao** [comment]
`stat` is also available on Linux. it's the flag `-x` that doesn't work. Is there a equivalent that works on Linux?

**Comment by cht8687** [comment]
@stevemao  could you supply the description of -x opion in macOS?

**Comment by stevemao** [comment]
```
-x      Display information in a more verbose way as known from some Linux distribu-
             tions.
```

Also attaching `man stat` on macOS incase you are interested.

[stat-man.txt](https://github.com/you-dont-need/You-Dont-Need-GUI/files/733799/stat-man.txt)

**Comment by ashleyharvey** [comment]
> `stat` is also available on Linux. it's the flag `-x` that doesn't work. Is there a equivalent that works on Linux?

The output of plain GNU `stat` on Linux is almost the same as BSD's `stat` on macOS. Made the change in my PR.

**Comment by stevemao** [comment]
Thanks @ashleyharvey

**Pull request closed without merging**